src/assets/styles/app.css

```diff
@@ -0,0 +1,578 @@
+:root {
+    --primary-color: #142630;
+    --secondary-color: #e8e8e8;
+    --footer-bgcolor: #272727;
+    --sidebar-bgcolor: #5E5E5E;
+    --main-bgcolor: #9C9C9C;
+    /* --font-heading: Kalam, cursive, sans-serif; */
+    --font-heading: "Arial";
+    --font-text: Roboto sans-serif;
+    --font-fa-solid: "Font Awesome 6 Free";
+    --contrast: #fff9eb;
+}
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 16px;
+}
+
+body {
+    padding: 0;
+    margin: 0;
+    max-width: 100%;
+    font-family: var(--font-heading);
+    /* word-spacing: -0.15rem; */
+}
+
+img {
+    max-width: 100%;
+    display: block;
+}
+
+.logo {
+    align-self: center;
+    padding-left: 0.5rem;
+}
+
+.logo img {
+    max-height: 60px;
+    margin-bottom: 5px;
+    margin-top: 5px;
+    display: block;
+}
+
+.navigation {
+    display: flex;
+    justify-content: space-between;
+}
+
+.sign_tab {
+    align-self: center;
+    padding-right: 1rem;
+}
+
+.sign_tab img {
+    height: 1.5rem;
+    display: block;
+}
+
+.sign_tab .icon {
+    font-size: 2rem;
+    color: var(--secondary-color);
+}
+
+/* .icon {
+    color: var(--secondary-color);
+} */
+
+.sidebar_icon_link .icon {
+    font-size: 1.2rem;
+}
+
+h1 {
+    color: var(--primary-color);
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: var(--font-heading);
+    margin: 0;
+}
+
+.layout {
+    display: grid;
+    width: 100vw;
+    height: 100vh;   
+}
+
+.layout.no-sidebar {
+    grid-template-columns: 1fr;
+    grid-template-rows: 70px 1fr 50px;
+    grid-template-areas: 
+    "header"
+    "main_content"
+    "footer ";
+}
+
+.layout {
+    grid-template-columns: 4.5rem 1fr;
+    grid-template-rows: 70px 1fr 50px;
+    grid-template-areas: 
+    "header header"
+    "sidebar main_content"
+    "footer footer ";
+}
+
+.header {
+    grid-area: header;
+    background-color: var(--primary-color);
+    position: sticky;
+    z-index: 99;
+    /* box-shadow: 0.01rem 0 0.7rem var(--secondary-color); */
+    border-bottom: 1px solid var(--secondary-color);
+}
+.sidebar {
+    grid-area: sidebar;
+    background-color: var(--sidebar-bgcolor);
+    position: sticky;
+    /* border-right: 1px solid var(--secondary-color); */
+}
+
+.sidebar:empty {
+    display: none;
+}
+
+.main_content {
+    grid-area: main_content;
+    background-color: var(--main-bgcolor);
+    padding-left: 1rem;
+    padding-right: 1rem;
+    border-left: 1px solid var(--secondary-color);
+}
+.footer {
+    grid-area: footer;
+    background-color: var(--footer-bgcolor);
+    position: sticky;
+    border-top: 1px solid var(--secondary-color);
+}
+
+.useful_links {
+    list-style: none;
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+}
+
+.useful_links a {
```

Comment:
some selectors use `_` (underscores), while others use `-` (dashes) - it would be better if only one would be used